ethereum/spec/src/main/java/tech/pegasys/teku/spec/config/SpecConfigElectra.java

```diff
@@ -67,6 +67,10 @@ static SpecConfigElectra required(final SpecConfig specConfig) {
 
   UInt64 getFieldElementsPerCell();
 
+  default UInt64 getNumberOfColumns() {
+    return UInt64.valueOf(getFieldElementsPerBlob()).times(2).dividedBy(getFieldElementsPerCell());
```

Comment:
Maybe worth to add also `getFieldElementsPerExtBlob()` with `getFieldElementsPerBlob() * 2`, so it will be more clear and somebody wil not take old one when new one is needed